modules/client-configuration/pages/channels.adoc

```diff
@@ -21,17 +21,31 @@ For example, if a system has been assigned to the {sles}{nbsp}15 `x86_64` base c
 In the {productname} {webui} you can browse your available channels by navigating to menu:Software[Channel List > All].
 You can modify or create new channels by navigating to menu:Software[Manage > Channels].
 
+On Salt clients, you must apply the highstate after subscribing to the channel to be able to install packages.
+
 For more on using channels, including custom channels, see xref:administration:channel-management.adoc[].
 
 
 
 == Packages Provided by {phub}
 
 {phub} is an extension to {sle} products that provides additional open source software provided by the {opensuse} community.
+
+[IMPORTANT]
+====
+The packages in {phub} are provided by the {opensuse} community.
+They are not supported by {suse}.
+====
+
 If you are using {sle} operating systems on your clients, you can enable the {phub} extension to access these additional packages.
 This provides the {phub} channels, which you can subscribe your clients to.
 
-On Salt clients, you must apply the highstate after subscribing to the channel to be able to install packages.
+{phub} provides a large number of packages, which can take a long time to synchronize and consume a large amount of disk space.
+Do not enable {phub} unless you require the packages it provides.
+
+To avoid unintentionally installing or updating unsupported packages, we recommended that you implement a content lifecycle management strategy that initially denies all {phub} packages.
```

Comment:
```suggestion
To avoid unintentionally installing or updating unsupported packages, we recommend that you implement a content lifecycle management strategy that initially denies all {phub} packages.
```

Sorry, for some reason I used the past tense in my previous suggestion :-/

Reply:
I really should have noticed that :woman_facepalming: